src/defaults/settings.js

```diff
@@ -49,6 +51,17 @@ export default //Renderer-specific settings
             }
         }
     ],
+    colors: [
```

Comment:
Is this resolving #34? If so, lets move that issue to the 0.2 project and milestone